File: sentimental-mario-less/mario.py
from cs50 import get_int

height = get_int("Height :")

while height < 1 or height > 8:
    print("chouse between 1 and 8.")
    height = get_int("Height :")

if height > 0 or height < 9:
    space = height
    for line in range(height):

        for i in range(space - 1):
            print(" ", end="")

        for j in range(line + 1):
            print("#", end="")

        space -= 1
        print("")

File: practice/more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int t;
    int f = 1 ;
    int s = 0 ;

    do
    {
         t =get_int("height :  ");
    }
    while ( t < 1 || t > 8);

      for(int h = 0 ; h < t ; h++ )
        {
            for(int p = 7 ; p > s  ; p-- )
            {
               printf(" ");
            }

            for(int w = 0 ; w < f  ; w++ )
            {
                  printf("#");

            }

            printf("   ");

            for(int w = 0 ; w < f  ; w++ )
            {
                  printf("#");

            }


          f++;
          s++;
          printf("\n");
        }

}